src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -31,100 +31,114 @@ function ProductGallery({ title }: Props) {
       return
     }
 
-    // notebook breakpoint = 1280px (See breakpoints on styles/global.scss)
-    setIsMobile(screenWidth < 1280)
+    const notebookBreakpoint =
+      getComputedStyle(document.documentElement).getPropertyValue(
+        '--breakpoint-notebook'
+      ) || '1280'
+
+    setIsMobile(screenWidth < parseInt(notebookBreakpoint, 10))
   }, [screenWidth])
 
   if (!data) {
-    return <div className="temp-data-loading">loading...</div>
+    return <div className="plp-temp-data-loading">loading...</div>
   }
 
   return (
-    <>
-      {/* Filters */}
-      <div>
+    <div className="plp-content-grid / grid-content">
```

Comment:
We don't have horizontal paddings on mobile. I would suggest using `grid-content-full` in this case. But on desktop, we still need `grid-content`. _Maybe_ , since we have `isMobile` variable here... we could use `isMobile ? 'grid-content-full' : 'grid-content'` or we can just overwrite the CSS.

![image](https://user-images.githubusercontent.com/3356699/150370470-cc2646cf-9ba3-43c1-befd-80ce93adda81.png)